website/docs/r/cloud_aws_link_account.html.markdown

```diff
@@ -0,0 +1,46 @@
+---
+layout: "newrelic"
+page_title: "New Relic: newrelic_cloud_aws_link_account"
+sidebar_current: "docs-newrelic-resource-cloud-aws-link-account"
+description: |-
+    Link an AWS account to New Relic.
+---
+
+# Resource: newrelic\_cloud\_aws\_link\_account
+
+Use this resource to link an AWS account to New Relic.
+
+## Prerequisite
+
+Setup is required in AWS for this resource to work properly. Complete the [steps outlined here](https://docs.newrelic.com/docs/infrastructure/amazon-integrations/get-started/connect-aws-new-relic-infrastructure-monitoring#connect) before using this resource.
```

Comment:
We need to also add this option: https://docs.newrelic.com/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-metric-stream/#set-up-metric-stream here.